services/ens/database.go

```diff
@@ -8,21 +8,29 @@ type Database struct {
 	db *sql.DB
 }
 
-type UsernameDetails struct {
+type UsernameDetail struct {
```

Comment:
Surely this is a minor change, but does it make sense? It's not just 1 detail inside 🤔

Reply:
make sense when something like `var usernameDetails []*UsernameDetail` comes to me

> It's not just 1 detail inside
if you mean username|chainId are details, IMO, username/chainId could belongs to UsernameDetail, each of them is part of UsernameDetail. @igor-sirotin